src/Auth/Identifiable.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Auth;
+
+interface Identifiable
+{
+    public function source(): string;
+
+    public function identifier(): string;
```

Comment:
Maybe rename to `identifierField`, but I don't have strong opinion about it.

Reply:
I think that’s a good idea, I’ll do it!